modules/@apostrophecms/ui/ui/apos/components/AposSlatList.vue

```diff
@@ -65,12 +61,6 @@ export default {
     selected: {
       type: String,
       default: null
-    },
-    field: {
```

Comment:
There is no field. This prop isn't used anymore, only serves to show the message, the message has been replaced in design, and the field is not passed in in any use of the thing.